packages/http/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @turnkey/http
 
+## 2.19.0
+
+### Minor Changes
+
+- 57f9cb0: Update endpoints - surface GetWalletAccount
```

Comment:
tiny nit: if you can add the mono version here as well that would be helpful for future reference 🙏